project-progression-template.md

```diff
@@ -4,24 +4,47 @@
 3.  For any questions or comments on this process please email info@confidentialcomputing.io
 
 ## Project Proposal Template
-1.  Name of Project
-2.  Project Description (what it does, why it is valuable, origin and history)
-3.  How does this project align with the Consortium's [Mission Statement](README.md)
-4.  High level assessment of project synergy with existing projects under the CCC, including how the project compliments/overlaps with existing projects, and potential ways to harmonize over time. Responses may be included both inline and/or in accompanying documentation.
-5. Describe the [Trusted Computing Base (TCB)](https://en.wikipedia.org/wiki/Trusted_computing_base) of the project. If the project supports multiple environments please describe each TCB. Also identify if dependencies of other project (both CCC or non-CCC) TCBs are taken.
-6.  Project website URL
-7.  Project Code of Conduct URL
-8.  Source control URL
-9.  Issue tracker URL
-10.  Project Logo URL or attachment (Vector Graphic: SVG, EPS)
-11. Project license
-12. External dependencies (including licenses)
-13. Standards implemented by the project, if any. Include links to any such standards.
-14. Release methodology and mechanics
-15. Names of initial committers, if different from those submitting proposal
-16. List of project's official communication channels (slack, irc, mailing lists)
-17. Social media accounts
-18. Existing financial sponsorship
-19. Trademark status
-20. Preferred maturity level (Sandbox, Incubation, Graduation, or Emeritus)
-21. Any additional information the TAC and Board should take into consideration when reviewing your proposal.
+All fields required. Throughout this document, the short answers (280 characters or less) are required; extended answers are encouraged but optional.
+### Project Information
+1.  Current name of project and any public repos/websites, social media, or chat channels
+2.  Project point of contact (name and email address)
+3.  Current project license
+4.  If donated to the Consortium, will this project be licensed differently?
+5.  Does your project currently have a CLA? If so, please link to the documentation of your CLA.(Using the Developer Certificate of Origin is considered a CLA.)
+6.  Have you read the Consortium’s project acceptance process and understand the IP and trademark statuses of project’s donated to the Consortium? [LINK] (A “no” answer is okay; the committee can answer questions about the policy.)
```

Comment:
@spjohnso I liked this in the form of a question because it lets the TAC/legal committee know that the project has questions, rather marking "I have read" but maybe not fully digesting what it means

Reply:
Comment here
```suggestion
6.  Have you read the Consortium’s project acceptance process and understand the IP and trademark statuses of project’s donated to the Consortium? More here. [LINK] (A “no” answer is okay; the committee can answer questions about the policy.)
```

Reply:
that's fine with me